src/main/java/seedu/address/logic/commands/NextCommandHistoryCommand.java

```diff
@@ -0,0 +1,39 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+
+/**
+ * Navigate to the next command. Detected via down-arrow key.
+ */
+public class NextCommandHistoryCommand extends Command {
+    public static final String COMMAND_WORD = "downCommand";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Views the next command entered. ";
+
+    public static final String MESSAGE_SUCCESS = "Next command is shown";
+    @Override
```

Comment:
Code style: consider to remove line 13 and 15, and add new line between line 16 and 17

Reply:
Sounds good to me thanks